### ISSUE TITLE
Use to SCP from local host to remote?

### ISSUE THREAD

**Issue by JLLeitschuh** [jcabi/jcabi-ssh#36]
Is it possible to use this library to SCP a file from your local machine to a remote machine?

The only library that seems to do this well is [JSch](http://www.jcraft.com/jsch/examples/ScpTo.java.html) which seems like a beast of a library to try to understand and use.

**Comment by dmarkov** [comment]
@yegor256 please pay attention to this issue ([par.21](http://at.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@JLLeitschuh no, it's not possible to do use SCP with jcabi-ssh, but this is how we're copying files, in another project, using jcabi-ssh: https://github.com/yegor256/rultor/blob/master/src/main/java/com/rultor/agents/daemons/Script.java#L65-L73
